ckanext/datitrentinoit/profile.py

```diff
@@ -0,0 +1,64 @@
+  
+import logging
+
+from rdflib.namespace import Namespace, RDF, SKOS
+from rdflib import URIRef, BNode, Literal
+
+import ckan.logic as logic
+
+from ckanext.dcat.profiles import RDFProfile, DCAT, LOCN, VCARD, DCT, FOAF, ADMS, OWL, SCHEMA, TIME
+from ckanext.dcatapit.dcat.profiles import remove_unused_object
+
+DCATAPIT = Namespace('http://dati.gov.it/onto/dcatapit#')
+
+log = logging.getLogger(__name__)
+
+
+class DatitrentinoitProfile(RDFProfile):
+    """
+    Customization of DCATAP-IT for Regione Umbria
+    """
+
+    def parse_dataset(self, dataset_dict, dataset_ref):
+        return dataset_dict
+
+    def graph_from_dataset(self, dataset_dict, dataset_ref):
+        g = self.g
+        org_dict = _get_org(dataset_dict.get('owner_org'))
```

Comment:
What do you need this info for? Is it used?

Reply:
Done